### ISSUE TITLE
What would cause this mod to not work?

### ISSUE THREAD

**Issue by DarkClippy** [ineedbots/iw4x_bot_warfare#104]
The installation instructions seem very, very simple. And I'm not using any other mods, but when I launch this mod after installing nothing happens. I can't even open the menu.

**Comment by BruceW1939** [comment]
Same issue. I successfully launch the mod as I can see in the top left corner "Mods: mp_bots" is active yet when I launch the game I can't open the menu. I have tried with an Xbox and Playstation controller as well. Nothing will open the mod menu.

**Comment by ghost** [comment]
did ya guys activate console on ur game? coz in order to make the mod work, u gotta press "5", then the mod menu will show up and u can set the amount of bots. if u aint have console activated, presing "5" will do nothing. the other way is editing the "zvr_bots" file so the bots spawn automatically without u accesing the mod menu.

**Comment by SRC267** [comment]
> did ya guys activate console on ur game? coz in order to make the mod work, u gotta press "5", then the mod menu will show up and u can set the amount of bots. if u aint have console activated, presing "5" will do nothing. the other way is editing the "zvr_bots" file so the bots spawn automatically without u accesing the mod menu.

Which part in zvr_bots needs the automatic flag? I could do with this on my handhelds, for convenience sake

**Comment by ghost** [comment]
Here, this is the route: z_svr_bots.iwd\maps\mp\bots, in there u'll find a file named "bot.gsc", there u must find a section called "bots_manage_fill", next to it, write the number of bots u wanna add to the match (i.e. for a 6v6 match, u'd need 11 bots, and so on..).

Edit: My bad, in that section u must add the amount of bots according to how many players u want in the match in total.. so u must add "12" instead of "11" for a 6v6 match, when u log in, u'll replace one of the bots.. you could also use the section "bots_manage_add", but that one only works for a couple of matches, the "fill" options works always.

**Comment by SRC267** [comment]
> Here, this is the route: z_svr_bots.iwd\maps\mp\bots, in there u'll find a file named "bot.gsc", there u must find a section called "bots_manage_fill", next to it, write the number of bots u wanna add to the match (i.e. for a 6v6 match, u'd need 11 bots, and so on..).
> 
> Edit: My bad, in that section u must add the amount of bots according to how many players u want in the match in total.. so u must add "12" instead of "11" for a 6v6 match, when u log in, u'll replace one of the bots.. you could also use the section "bots_manage_add", but that one only works for a couple of matches, the "fill" options works always.

I followed this and it works! It's a nice option, I also set bot difficulty to 7 for hardest, and set good loadouts only to "true" no more bad weapon combinations.

I've also seen the bot_chat.gsc file, can we simply rename some of the bot chatter to anything? For e.g. one of the chats says "Teeeam deeaaathhhmaaaachhh!" I'd like to replace that. Is it that simple? I'll have to test it out. Thx

**Comment by DarkClippy** [comment]
> did ya guys activate console on ur game? coz in order to make the mod work, u gotta press "5", then the mod menu will show up and u can set the amount of bots. if u aint have console activated, presing "5" will do nothing. the other way is editing the "zvr_bots" file so the bots spawn automatically without u accesing the mod menu.

Hi! Thanks for the advice, sir. But how do we enable the console? I actually just ran IW4 for the first time I guess and got the launcher working again. 😅 But indeed, pressing 5 didn't do anything.

**Comment by ghost** [comment]
> > did ya guys activate console on ur game? coz in order to make the mod work, u gotta press "5", then the mod menu will show up and u can set the amount of bots. if u aint have console activated, presing "5" will do nothing. the other way is editing the "zvr_bots" file so the bots spawn automatically without u accesing the mod menu.
> 
> Hi! Thanks for the advice, sir. But how do we enable the console? I actually just ran IW4 for the first time I guess and got the launcher working again. 😅 But indeed, pressing 5 didn't do anything.

Enabling console is easy, just go to the options menu on the game, there ur gonna find a section called "gameplay options" or "multiplayer options", i cant recall it now.. in there u'll find an option called "enable console", simple as that, just put "yes" and ur set.

**Comment by pveenthusiast** [comment]
> > Here, this is the route: z_svr_bots.iwd\maps\mp\bots, in there u'll find a file named "bot.gsc", there u must find a section called "bots_manage_fill", next to it, write the number of bots u wanna add to the match (i.e. for a 6v6 match, u'd need 11 bots, and so on..).
> > Edit: My bad, in that section u must add the amount of bots according to how many players u want in the match in total.. so u must add "12" instead of "11" for a 6v6 match, when u log in, u'll replace one of the bots.. you could also use the section "bots_manage_add", but that one only works for a couple of matches, the "fill" options works always.
> 
> I followed this and it works! It's a nice option, I also set bot difficulty to 7 for hardest, and set good loadouts only to "true" no more bad weapon combinations.
> 
> I've also seen the bot_chat.gsc file, can we simply rename some of the bot chatter to anything? For e.g. one of the chats says "Teeeam deeaaathhhmaaaachhh!" I'd like to replace that. Is it that simple? I'll have to test it out. Thx

Oh, u mean the text chat from the bots? What I did was simply disabling the entire chat, since I play solo... to do that u gotta open console and type the command "cg_chatHeight", there u gotta assign a value so.. just put "0" and that's it. The chat still happens, but u just wont see it anymore..

**Comment by DarkClippy** [comment]
> > > did ya guys activate console on ur game? coz in order to make the mod work, u gotta press "5", then the mod menu will show up and u can set the amount of bots. if u aint have console activated, presing "5" will do nothing. the other way is editing the "zvr_bots" file so the bots spawn automatically without u accesing the mod menu.
> > 
> > 
> > Hi! Thanks for the advice, sir. But how do we enable the console? I actually just ran IW4 for the first time I guess and got the launcher working again. 😅 But indeed, pressing 5 didn't do anything.
> 
> Enabling console is easy, just go to the options menu on the game, there ur gonna find a section called "gameplay options" or "multiplayer options", i cant recall it now.. in there u'll find an option called "enable console", simple as that, just put "yes" and ur set.

I did not see that option in my game, lol. I checked the singleplayer and multiplayer options because I figured it would be there, but I never saw anything about enabling console. Does that mean I missed an installation step or something?

**Comment by DarkClippy** [comment]
Small Update: It seems like the mod is working despite the fact that I can't access the menu, but I'm only able to open the menu once I'm already in a game?? Is that how it's supposed to work?

**Comment by ghost** [comment]
> Small Update: It seems like the mod is working despite the fact that I can't access the menu, but I'm only able to open the menu once I'm already in a game?? Is that how it's supposed to work?

Yeah, the mod works that way, u can access to its menu only after selecting ur faction and ur loadout.. however.. if u know how to tweak stuff, u can actually set the mod so it automatically deploys the bots without u accessing the menu.. check the replies I gave to the other user, where I told him where to go and what to change.. those files can be opened by using notepad so dont worry.. there u can set how many bots and their difficulty level.. except for the "zvr_bots" file, that one u must open it using winrar, winzip or any other zip files manager.

**Comment by DarkClippy** [comment]
> > Small Update: It seems like the mod is working despite the fact that I can't access the menu, but I'm only able to open the menu once I'm already in a game?? Is that how it's supposed to work?
> 
> Yeah, the mod works that way, u can access to its menu only after selecting ur faction and ur loadout.. however.. if u know how to tweak stuff, u can actually set the mod so it automatically deploys the bots without u accessing the menu.. check the replies I gave to the other user, where I told him where to go and what to change.. those files can be opened by using notepad so dont worry.. there u can set how many bots and their difficulty level.. except for the "zvr_bots" file, that one u must open it using winrar, winzip or any other zip files manager.

Well, using the menu is a little easier for me. 😅

I believe I was thinking of the Bots mod for World at War where you could actually use the menu from the lobby.

I may consider doing the pre-setup method you described some other time, but I also really like messing with the bot difficulty and number of bots pretty regularly, so the menu may be the best method for me.
Sorry for the confusion! 😅

Hope some other newbies like myself may find this thread and can figure it out on their own. Thank you fort the help!

**Comment by ghost** [comment]
No problem.. and, the good thing is that.. u can change the mod files as many times as u want.. trust me, i did it plenty of times so I could find a sweet spot for the bots, not too easy but not too tryhard either.. in the files where u can set all of it u can also set if u want them to jumpshot, dropshot or any other sweaty manouver that's programmed on the mod.
